contentgrid-spring-data-support/src/main/java/com/contentgrid/data/support/auditing/v1/UserMetadata.java

```diff
@@ -0,0 +1,25 @@
+package com.contentgrid.data.support.auditing.v1;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonValue;
+import jakarta.persistence.Embeddable;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Embeddable
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class UserMetadata {
+    @JsonIgnore
+    private String id;
+
+    @JsonIgnore
+    private String namespace;
+
+    @JsonValue
+    private String name;
```

Comment:
This field should also be marked as json read only

Reply:
no longer needed if top level `AuditMetadata` field is marked as read-only